src/fabric/src/fabric_util.erl

```diff
@@ -326,6 +326,8 @@ is_partitioned(DbName0) when is_binary(DbName0) ->
 is_partitioned(Db) ->
     couch_db:is_partitioned(Db).
 
+validate_all_docs_args(DbName, Args) when is_list(DbName) ->
+    validate_all_docs_args(list_to_binary(DbName), Args);
```

Comment:
Also not covered by unit tests.

Reply:
Add a quick test for fabric all docs. The `DbName` as a list is more of a remsh script kind of a affordance. But since we supported it before, I thought it's good to keep the compatibility.